docs/sphinx_setup/api/api_reference.rst

```diff
@@ -10,13 +10,13 @@ API Reference
    :maxdepth: 2
    :hidden:
 
+   ie_python_api/api
    c_cpp_api/group__ov__cpp__api
    c_cpp_api/group__ov__c__api
-   ie_python_api/api
-
+   nodejs_api/nodejs_api.rst
 
 
-OpenVINO toolkit offers **APIs for Python, C, and C++** which share most features (C++ being the
+OpenVINO toolkit offers **APIs for Python, C++, C, and Node.js** which share most features (C++ being the
```

Comment:
I am not sure that Node.js should be in this list. Because Python, C++, C are programming language, but Node.js it's technology. I think here would be better to use JavaScript (Node.js) - to highlight that our JS support is for Node.js environment only.